package.json

```diff
@@ -2,51 +2,58 @@
   "name": "eslint-plugin-vue",
   "version": "3.0.0",
   "description": "Official ESLint plugin for Vue.js",
-  "engines": {
-    "node": ">=4"
-  },
   "main": "lib/index.js",
+  "scripts": {
+    "start": "npm run test:simple -- --watch --growl",
+    "test:base": "mocha tests --recursive",
+    "test:simple": "npm run test:base -- --reporter nyan",
+    "test": "nyc npm run test:base",
+    "lint": "eslint .",
+    "pretest": "npm run lint",
+    "preversion": "npm run update && npm test",
+    "postversion": "git push --follow-tags",
+    "update": "node ./tools/update-rules.js"
+  },
   "files": [
     "lib"
   ],
+  "homepage": "https://github.com/vuejs/eslint-plugin-vue#readme",
+  "keywords": [
+    "eslint",
+    "eslint-plugin",
+    "eslint-config",
+    "vue",
+    "vuejs",
+    "rules"
+  ],
+  "author": "Toru Nagashima (https://github.com/mysticatea)",
+  "contributors": [
+    "Michał Sajnóg <msajnog93@gmail.com> (http://github.com/michalsnik)"
+  ],
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/vuejs/eslint-plugin-vue.git"
+  },
+  "bugs": {
+    "url": "https://github.com/vuejs/eslint-plugin-vue/issues"
+  },
+  "engines": {
+    "node": ">=4"
+  },
   "peerDependencies": {
     "eslint": "^3.18.0"
```

Comment:
Today, ESLint 4.0.0 has been released. I'd like to include 4.x in peerDependencies to avoid installation failure.